### ISSUE TITLE
GameSDK on Mobile

### ISSUE THREAD

**Issue by hnguyen094** [discord/gamesdk-and-dispatch#110]
Not a bug, but a feature request! It would be nice to be able to have some basic functionality on Android & iOS, through some platform-compiled binaries (e.g. `.so` files for ARM or something). We're currently working on a multi-user experience that utilizes the Lobby APIs to connect various app clients to the same world instance. Having the ability to connect clients to the same lobby across platforms would be very intuitive.

**Expected workflow**
1) Add some precompiled libs/binaries for ARM (whether iOS or Android).
2) The SDK would work as expected on those platforms.

**Implementation specifics**
- Not important to the request, but maybe as a data point we're working with C# Unity.
- Would be perfectly fine if the mobile SDK launched an Android Intent/open some URI for the Discord instance to join a lobby, using the `clientID` & other relevant parameters as arguments. Unclear what the security implications are, but would love to discuss what challenges there might be.

**Comment by Jump-Suit** [comment]
Mobile Discord Client has no way of communicating to the Discord GameSDK if this was to go forward into production. 
So sadly, I don't believe this is possible.

**Comment by hnguyen094** [comment]
Not even as intent-based URIs (e.g. `discord://discord.gg/somegame?authdata=xxxx&lobby=yyyy`)? Is the mobile Discord client unable to communicate with the lobby creation/join backend or is there some other technical challenge?

**Comment by Jump-Suit** [comment]
Mobile Discord can't communicate no.. Sorry. https://discord.com/channels/613425648685547541/613425918748131338/619218346050846721
Read Mason's response about it here.
It is what it is.

**Comment by hnguyen094** [comment]
Yup, it makes sense! It seems like it's well discussed already so I won't go too much more into it. But at the very least it seems possible for lobbies through HTTP:
![image](https://user-images.githubusercontent.com/23398543/111821164-ba8b8d80-88b8-11eb-8a70-33ecda2feaaf.png)

Totally understand that
1) Devs might want to have full feature-parity between mobile & desktop, which is impossible architecturally. But having just lobbies on mobile alone would be fantastic! 
2) There simply isn't enough people wanting this, and means that this kind of feature would just get backlogged until enough people (e.g. me & you, this thread) express more interest for it.
